### PR TITLE
Update bower.json

### PR DESCRIPTION
**Pull request by zekida**
"main" field in bower.json should list all the files needed to load this element(like Polymer's bower.json)

### CODE REVIEW

**Comment by zekida** [comment]
I signed it!

**Comment by frankiefu** [comment]
[app-layout.html](https://github.com/PolymerElements/app-layout/blob/master/app-layout.html) already includes all the main elements so we don't need to add the files separately.

**Comment by zekida** [comment]
yes we do... that's why i specifically referred [Polymer's bower.json](https://github.com/Polymer/polymer/blob/master/bower.json) as an example.

bower does not parse through imports in 'app-layout.html'. Thats for the html renderer to handle.
the usual bower based frontend tooling pulls the 'main' field as required assets and pushes them to a development server or to a CDN if deploying.

**Comment by frankiefu** [comment]
@zekida Thanks for the detailed explanations.  Now we understand more about the issue and what this PR is trying to solve.  I reopened it and we will take a closer look.  So far it seems pretty reasonable to me.

p.s. In the future it would be good to file an issue so we could have a clear understanding of the issue: https://github.com/PolymerElements/app-layout/blob/master/CONTRIBUTING.md#submitting-pull-requests

**Comment by zekida** [comment]
@frankiefu Thank you for the quick reply, and the reconsideration!
apologies for the abrupt PR, next time I will open an issue beforehand :)

**Comment by abdonrd** [comment]
The paper-input already do that: [PolymerElements/paper-input/bower.json#L13-L21](https://github.com/PolymerElements/paper-input/blob/master/bower.json#L13-L21)

File: bower.json [hunk 1]
```diff
@@ -11,7 +11,19 @@
...
-  "main": "app-layout.html",
+  "main": [
+    "app-layout.html",
+    "helpers/helpers.html",
+    "app-drawer/app-drawer.html",
+    "app-drawer-layout/app-drawer-layout.html",
+    "app-grid/app-grid-style.html",
+    "app-header/app-header.html",
+    "app-header-layout/app-header-layout.html",
+    "app-scroll-effects/app-scroll-effects-behavior.html",
```
Comment (keanulee) [inline bower.json]: To be complete, each of the effects in app-scroll-effects/effects/, as well as app-scroll-effects.html, should be listed here

Comment (zekida) [inline bower.json]: agreed. sorry for the incomplete PR! thanks!

**Pull request closed without merging**